.github/workflows/wix.yml

```diff
@@ -69,38 +84,24 @@ jobs:
           MSI_PRODUCT_VERSION: ${{ matrix.MSI_PRODUCT_VERSION }}
           ARCH: ${{ matrix.ARCH }}
           JVM: ${{ matrix.JVM }}
-
-      - name: Download IcedTea-Web
-        run: |
-          cd wix\SourceDir
-          wget -q "https://github.com/AdoptOpenJDK/IcedTea-Web/releases/download/${{ matrix.ICEDTEA_WEB_VERSION }}/${{ matrix.ICEDTEA_WEB_VERSION }}.win.bin.zip"
-          unzip icedtea-web-*.win.bin.zip
-          rm icedtea-web-*.win.bin.zip
-          Remove-Item 'icedtea-web-image\share\doc' -Recurse
-        if: ${{ matrix.ICEDTEA_WEB_VERSION != '' }}
-
-      - name: Download Prebuilt JDK/JRE
-        run: |
-          cd wix\SourceDir
-          wget -q "https://github.com/adoptium/temurin${{ env.PRODUCT_MAJOR_VERSION }}-binaries/releases/download/${{ matrix.TAG }}/OpenJDK${{ env.PRODUCT_MAJOR_VERSION }}U-jdk_${{ env.ARCH }}_windows_${{ env.JVM }}_${{ matrix.SUB_TAG }}.zip"
-          wget -q "https://github.com/adoptium/temurin${{ env.PRODUCT_MAJOR_VERSION }}-binaries/releases/download/${{ matrix.TAG }}/OpenJDK${{ env.PRODUCT_MAJOR_VERSION }}U-jre_${{ env.ARCH }}_windows_${{ env.JVM }}_${{ matrix.SUB_TAG }}.zip"
-          ./CreateSourceFolder.AdoptOpenJDK.ps1
-
-      - name: Create JDK Installer
-        run: |
-          cd wix
-          set PRODUCT_CATEGORY=jdk
-          call Build.OpenJDK_generic.cmd
         shell: cmd
 
       - name: Create JRE Installer
-        run: |
-          cd wix
-          set PRODUCT_CATEGORY=jre
-          call Build.OpenJDK_generic.cmd
+        run: call Build.OpenJDK_generic.cmd
+        working-directory: wix
+        env:
+          PRODUCT_CATEGORY: jre
+          PRODUCT_MAJOR_VERSION: ${{ matrix.PRODUCT_MAJOR_VERSION }}
+          PRODUCT_MINOR_VERSION: ${{ matrix.PRODUCT_MINOR_VERSION }}
+          PRODUCT_MAINTENANCE_VERSION: ${{ matrix.PRODUCT_MAINTENANCE_VERSION }}
+          PRODUCT_PATCH_VERSION: ${{ matrix.PRODUCT_PATCH_VERSION }}
+          PRODUCT_BUILD_NUMBER: ${{ matrix.PRODUCT_BUILD_NUMBER }}
+          MSI_PRODUCT_VERSION: ${{ matrix.MSI_PRODUCT_VERSION }}
+          ARCH: ${{ matrix.ARCH }}
+          JVM: ${{ matrix.JVM }}
         shell: cmd
 
-      - uses: actions/upload-artifact@a8a3f3ad30e3422c9c7b888a15615d19a852ae32 # v3.1.3
+      - uses: actions/upload-artifact@c7d193f32edcb7bfad88892161225aeda64e9392 # v4.0.0
```

Comment:
Is that the latest version?

Reply:
it's to bring it up to date with the others. Dependabot will then do the next upgrade